handbook/engineering/code_reviews.md

```diff
@@ -67,6 +67,7 @@ Do:
   - Describe any remaining open questions that you are seeking feedback on.
 - Make it clear who's review(s) you are waiting for.
 - Politely remind your reviewer that you are waiting for a review if they haven't responded after one business day.
+- Don't force push commits that erase the latest reviewed commit or any commit prior to that. This makes it easy for reviewers to pick up where they left off in the previous round.
```

Comment:
Can we add that prior to requesting reviews or having recevied a review it's fine to clean up commits by force pushing?

I think this sounds much more general than intended. As I understood it, this rule should help avoid a situation where a PR goes through multiple rounds of reviews and the reviewers lose track of what they already looked at after the PR author force-pushed changes in between rounds. I think the rule is good to avoid _that situation_, but in other situations it's, IMHO, fine to force push for whatever reasons.

Reply:
@mrnugget: Sure thing, please suggest how you'd like to word it.

Reply:
https://github.com/sourcegraph/about/pull/430/files#r353808172